sql/moz-fx-data-shared-prod/monitoring_derived/structured_distinct_docids_v1/query.py

```diff
@@ -61,7 +60,8 @@
   submission_date
 """
 
-EXCLUDED_NAMESPACES = {"xfocsp_error_report"}  # restricted access
+# restricted access
+EXCLUDED_NAMESPACES = {"xfocsp_error_report", "contextual_services"}
```

Comment:
Should this matter for monitoring queries on authorized views? One of these is actually a doctype under the telemetry namespace so maybe was ignored or was supposed to be `regrets-reporter`.

Reply:
I think Airflow will have necessary privs to access all of these; not sure why this was included originally. Maybe this is totally unnecessary now.